tests/integration/test_schema.py

```diff
@@ -1250,48 +1396,77 @@ async def test_schema(registry_async_client, trail):
                 "name": "first_name",
                 "type": "string",
             },
+            {
+                "name": f"{unique}",
+                "type": "string",
+            },
             {
                 "name": "last_name",
                 "type": "string",
             },
-        ]
+        ],
+        "unique": f"{unique_field_factory()}"
```

Comment:
```suggestion
        "unique": unique_field_factory()
```
The result of `unique_field_factory()` should already be a string. This also applies to the usages below

Reply:
Fixing.